dir.props

```diff
@@ -136,14 +136,15 @@
     <_portableOS>linux</_portableOS>
     <_portableOS Condition="'$(_runtimeOSFamily)' == 'win'">win</_portableOS>
     <_portableOS Condition="'$(_runtimeOSFamily)' == 'osx'">osx</_portableOS>
+    <_portableOS Condition="'$(_runtimeOSFamily)' == 'FreeBSD'">freebsd</_portableOS>
```

Comment:
Is the the correct casing that comes out of the os-release file?

Reply:
I check logs and OSFamily is FreeBSD and OS is FreeBSD.11 in my case. 
However the generated rid needs to be all lowercase to match coreclr.

Reply:
> msbuild does case sensitive string compares 

I always thought MSBuild is case insensitive.

```xml
<Project DefaultTargets="Build">
  <PropertyGroup>
    <ConditionsAre>case sensitive</ConditionsAre>
    <ConditionsAre Condition="'lower' == 'LOWER'">case insensitive</ConditionsAre>
  </PropertyGroup>

  <Target Name="Build">
    <Message Text="ConditionsAre is $(ConditionsAre)" Importance="High" />
  </Target>
</Project>
```

Output

```
> dotnet msbuild
Microsoft (R) Build Engine version 15.8.138-preview+ge0cd4f1c1f for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  ConditionsAre is case insensitive
```

Reply:
I've always made the opposition assumption, especially with cross-plat. I could be mixing it up with all the case sensitive issues we have hit around the filesystem.

Reply:
Either way, IMO it is good practice to match casing when possible.  So if `/etc/os-release` has it with upper-case characters, I would match it.